githubflow_release/release.py

```diff
@@ -150,7 +150,18 @@ def _get_merged_pullrequest(self):
             # and if distant/release contains HEAD of PR
             # (stops after 10 successive merged PR)
             if pr.is_merged:
-                branches = self.git.branch('-r', '--contains', pr.head_sha1) + '\n'
+                try:
+                    branches = self.git.branch('-r', '--contains', pr.head_sha1) + '\n'
+                except GitCommandError as e:
+                    # if a PR
+                    #     - a PR is removed by a reset --hard and a push --force
+                    #     - a PR is squashed and merged --fast-foward
+                    # the git history is rewritten
+                    # so pr.head_sha1 does not exist anymore in the git history
+                    # Therefore pr.title will not appear in the changelog
+                    logging.warning("Commit {} of PR {} not found".format(pr.head_sha1, pr.url))
+                    continue
+            self.repo.create_head(RELEASE_BRANCH)
```

Comment:
Not sure I understand why you need to add that line though :thinking:

Reply:
oups